streamer/Cargo.toml

```diff
@@ -63,7 +63,9 @@ tokio-util = { workspace = true }
 x509-parser = { workspace = true }
 
 [dev-dependencies]
+anyhow = { workspace = true }
 assert_matches = { workspace = true }
+clap = { version = "4.5.31", features = ["cargo", "derive", "error-context"] }
```

Comment:
can't we have this dependency on the workspace level called like `clap-v4`? I'm afraid that we might end up having many dependencies with difference in patch version.

Reply:
Due to how clap derive works it is not actually possible to have it under a different crate name, as it breaks the derive macros. I've tried this already before, maybe my cargo-fu is not strong enough though.